packages/ember/tests/routing/substates_test.js

```diff
@@ -415,6 +415,41 @@ QUnit.test('Default error event moves into nested route', function() {
   equal(appController.get('currentPath'), 'grandma.error', 'Initial route fully loaded');
 });
 
+QUnit.test('Error events that aren\'t bubbled don\t throw application assertions', function() {
+  expect(2);
```

Comment:
also, note: When this run under 2.1, three assertions will be fired, one of which will be failed. That's the thrown error from the non-bubbled error hook. This didn't happen in any previous ember version.